### ISSUE TITLE
It will be good to be able to rearrange the avatars the way the user wants

### ISSUE THREAD

**Issue by BanuPavithraV** [mattriley/agile-avatars#9]
Currently we have the ability to sort the avatars in alphabetic/reverse-alphabetic order. It will be good to have options to move the avatars around for logical grouping

**Comment by mattriley** [comment]
Hi Banu. Thanks for your suggestion! Could you help me understand this use case a little better? My feeling was that sorting was not particularly valuable given the idea is to print and cut them out.

**Comment by BanuPavithraV** [comment]
Hi Matt, that was super quick :).

I used the application for a different use - I created our team collage using it, this was to highlight team   who are part of a release that we are just about to announce. While I could create the collage in moments, I had to spend time to align it - Hence the request. 

Thanks,
Banu

- [ ]

**Comment by mattriley** [comment]
How interesting! I was actually thinking to make a similar tool for this purpose but I can see how this could serve a dual purpose with some minor tweaks. Would it be helpful to have the option to drop the role onto the next line to give it more space?

**Comment by BanuPavithraV** [comment]
Yes, bringing the role in next line will most likely fix the alignment issues. 

In the above pic, I wanted to align Chao, Ke Zhou & Tianran together, that's where I felt that it may be cool to have the align options given to the user

Also, another suggestion - "The name and role will be extracted from the filename. To save time on editing, rename images to name[+role].ext, e.g. matt.jpg or matt+dev.jpg. Roles are not limited to the list displayed. All names and roles can be edited."  It will be probably good to add is in the landing page as well on top.

**Comment by mattriley** [comment]
Hi Banu. I've made a couple of changes that might help.

If you prefix the filenames with numbers, they will sort naturally in your file explorer/finder. Then when you drag them in, they'll be added in that order (this is existing functionality). The change I've made is that any number prefix on the filename will be stripped out. Please give it a try if you have a moment. It's a "hidden" feature at this time - I'd love to add a drag n drop sort but that'll have to be later.

I've also added the (reworded) file naming tip you mentioned, with the ability to dismiss it.

I'm also intending to provide an option to drop the role down to the next line.

**Comment by BanuPavithraV** [comment]
Hi Matt, I did try number prefix and it works cool! I observe that you have added the file naming tip text (with dismiss option) as well. 

Thanks for considering my feedback and working on the same.

Cheers,
Banu
